src/odemis/driver/tescan.py

```diff
@@ -1127,12 +1160,25 @@ def _doMoveAbs(self, pos):
             logging.debug("Stage move to %s completed", pos)
             self._updatePosition()
 
-    def _doMoveRel(self, shift):
-        # Compute the position in absolute coordinates
+    def _doMoveRel(self, shift: dict):
+        """
+        Request the position in absolute coordinates and create new positions after applying shift values
+        :param shift: shift of linear axes in m or rotational axes in radians
+        """
         self._updatePosition()
         pos = self._position.copy()
+
+        # add the requested change to the current position
         for axis, change in shift.items():
             pos[axis] += change
+            if axis in ("rx", "rz"):
+                rng = self.axes[axis].range
+                # check if any of the range values are 360°, range values for full rotational axis
+                # requested with command StgGetLimits() always returns (-360, 360)
+                if (2 * math.pi) in [abs(ele) for ele in rng]:
+                    # this check is for full rotational axes to handle under- and overflow
```

Comment:
This comment is not completely clear, if I read the code correctly it maps values outside of the range to between 0 and 2pi? Then I would say something like
```suggestion
                    # for full rotational axes this check maps values outside the range to between 0 and 2pi
```